keyboards/naked64/keymaps/default/keymap.c

```diff
@@ -0,0 +1,157 @@
+  //|--------+--------+--------+--------+--------+--------+--------|             |--------+--------+--------+--------+--------+--------+--------|
+               _______,   KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,                 KC_F6, KC_MINS,  KC_EQL, KC_LBRC, KC_RBRC, KC_BSLS, _______, \
+  //         |--------+--------+--------+--------+--------+--------|             |--------+--------+--------+--------+--------+--------+--------|
+               _______,   KC_F7,   KC_F8,   KC_F9,  KC_F10,  KC_F11,                KC_F12, KC_NUHS, KC_NUBS, _______, _______, _______, _______, \
+  //         |--------+--------+--------+--------+--------+--------|             |--------+--------+--------+--------+--------+--------+--------|
+               _______,          _______,          _______, _______,               _______,          _______, _______, _______, _______, _______  \
+          //`-----------------------------------------------------------------------------------------------------------------------------------'
+  ),
+
+  [_ADJUST] = LAYOUT( /* Base */
+  //,--------------------------------------------------------------|             |--------------------------------------------------------------.
+   TG(_FLOCK),   RESET,   KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,                 KC_F6,   KC_F7,   KC_F8,   KC_F9,  KC_F10,  KC_F11,  KC_F12, \
+  //|--------+--------+--------+--------+--------+--------+--------|             |--------+--------+--------+--------+--------+--------+--------|
+      _______, _______, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,               RGB_RST, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, _______, \
+  //|--------+--------+--------+--------+--------+--------+--------|             |--------+--------+--------+--------+--------+--------+--------|
+               _______, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,               RGB_TOG, RGB_MOD, XXXXXXX,LCA(KC_DEL),LALT(KC_PSCR),KC_PSCR,XXXXXXX, \
+  //         |--------+--------+--------+--------+--------+--------|             |--------+--------+--------+--------+--------+--------+--------|
+               _______, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,               RGB_VAD, RGB_VAI, RGB_HUD, RGB_HUI, RGB_SAD, RGB_SAI, XXXXXXX, \
+  //         |--------+--------+--------+--------+--------+--------|             |--------+--------+--------+--------+--------+--------+--------|
+               _______,          _______,          _______, _______,               _______,          _______, _______, _______, _______, _______  \
+          //`-----------------------------------------------------------------------------------------------------------------------------------'
+  )
+};
+
+
+//A description for expressing the layer position in LED mode.
+uint32_t layer_state_set_user(uint32_t state) {
+#ifdef RGBLIGHT_ENABLE
+    switch (biton32(state)) {
```

Comment:
As with above, not always a 32 bit bitmask.  `get_highest_layer` will use the correct function based on the bitmask size:

```suggestion
    switch (get_highest_layer(state)) {
```